browse/templates/abs/extra_services.html

```diff
@@ -56,14 +56,10 @@
       {%- endif -%}
       {%- if format == 'src' and current-%}
   <li><a href="{{url_for('src.src', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
-      {%- elif format.startswith('ps') -%}
```

Comment:
As far as I can tell from ARXIVCE-1159 this PR isn't suppose to remove the 'ps' format from the list.

Reply:
Mark tells me this PR is intended to remove the PS download.

Reply:
We just had a quick chat (me and Mark) and were in agreement that PS should be found on the /formats page, but not on this main abs list.  Same with DVI.

Reply:
@jimentwood The PS download links on the abs page have been changed.